packages/venia-ui/lib/components/ProductImageCarousel/carousel.css

```diff
@@ -7,17 +7,23 @@
     grid-template-rows: auto;
 }
 
-.imageContainer {
+.carouselContainer {
     display: grid;
     grid-area: main;
     grid-template-columns: repeat(3, 1fr);
 }
 
+.imageContainer {
+    grid-area: 1 / 1 / 2 / 4;
+}
+
 .currentImage {
-    background-color: rgb(var(--venia-background-color));
-    border-radius: 2px;
     width: 100%;
-    grid-area: 1 / 1 / 2 / 4;
+}
+
+.currentImage_placeholder {
+    composes: currentImage;
+    background-color: rgb(var(--venia-grey));
```

Comment:
Can we just use the background color on `currentImage` and do away with this class entirely?

Reply:
I tried this and got the dreaded gray bar 😨 

![Screen Shot 2019-10-25 at 9 32 51 AM](https://user-images.githubusercontent.com/13182778/67580154-73bd5680-f70b-11e9-8e1f-b2309e760eac.png)

But I have no idea why, honestly. The `img` element boundary definitely does not include that area:

![Screen Shot 2019-10-25 at 9 40 56 AM](https://user-images.githubusercontent.com/13182778/67580230-9bacba00-f70b-11e9-9d98-af62acae6906.png)

And it is coming from this property - toggling it on and off in dev tools made it appear / disappear 🤷‍♂

Reply:
Well in the past that was happening because the placeholder image, which is rendered just behind the actual product image, was *slightly* bigger. I had fixed this by cropping/stretching the product images to cover the placeholder but it seems like that got lost somewhere.

I think with some investigation we could figure it out but probably not really worth it in this PR anymore.